### PR TITLE
Upgrade to primer 6

### PR DESCRIPTION
**Pull request by wilfriedE**
Using this PR to track work on resolving #921

### CODE REVIEW

**Comment by nwoodthorpe** [comment]
It looks like you're based on Marks Webpack PR somehow, can you fix that?

Lmk when this is good for :eyes: :+1:

**Comment by nwoodthorpe** [comment]
Looks like the mobile version isn't responsive anymore

**Comment by nwoodthorpe** [comment]
Organization name in the header is different between master and this

**Comment by nwoodthorpe** [comment]
Assignment titles in the list of assignments for a classroom are really large in this branch

**Comment by nwoodthorpe** [comment]
The hover state of the action buttons is a bit off
screen shot 2017-06-16 at 12 06 55 pm

**Comment by nwoodthorpe** [comment]
We lost the text padding on the `Reset and remove this classroom` dialog under `manage your classroom`

**Comment by nwoodthorpe** [comment]
Just took a look through each view, ^ that stuff is the only stuff I noticed. Most are small fixes, the only big thing might be fixing the mobile-friendly-ness of the app.

Ping me when you want :eyes: again!

File: app/assets/stylesheets/components/buttons.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// Variables
```
Comment (nwoodthorpe) [inline app/assets/stylesheets/components/buttons.scss]: Can we pull this from `variables.scss`?

Comment (wilfriedE) [inline app/assets/stylesheets/components/buttons.scss]: Yes,  😅  @nwoodthorpe  thanks for catching that.

File: package.json [hunk 2]
```diff
@@ -2,17 +2,40 @@
   "name": "classroom",
   "private": true,
   "dependencies": {
+    "autoprefixer": "^7.1.1",
```
Comment (tarebyte) [inline package.json]: Let's remove the dependencies we don't need here.

**Comment by nwoodthorpe** [comment]
Looks like something broke here:
screen shot 2017-06-19 at 1 04 27 pm

**Comment by nwoodthorpe** [comment]
The "Choose an existing set of teams" is still the old format

**Comment by nwoodthorpe** [comment]
The "Create Assignment" button on the group assignment / new form is stuck as grayed out

**Comment by nwoodthorpe** [comment]
The Repo URL here is off centered 
screen shot 2017-06-19 at 1 08 41 pm

**Comment by nwoodthorpe** [comment]
I noticed a few more things that are graphically broken, all small things though :)

**Comment by nwoodthorpe** [comment]
Mobile looks 10/10 now!

File: package.json (hunk 2 above)
Comment (wilfriedE) [inline package.json]: @tarebyte cutting it down to these dependencies. Anything that should be added/removed?
```
    "clipboard": "^1.7.1",
    "dragula": "3.7.1",
    "include-fragment-element": "^3.0.0",
    "jquery": "^3.2.1",
    "jquery-readyselector": "git+https://git@github.com/Verba/jquery-readyselector.git",
    "jquery-ujs": " ~1.2.2",
    "jquery.turbolinks": "git+https://git@github.com/kossnocorp/jquery.turbolinks#2.1.0",
    "primer-css": "^6.0.0",
    "rails-ujs": "^5.1.1",
    "remodal": "^1.1.1",
    "transliteration": "^1.6.2",
    "webcomponents.js": "^0.7.24",
    "whatwg-fetch": "^2.0.3"
```

Comment (tarebyte) [inline package.json]: Those should all stay for now 👍

**Pull request merged**